pagopa/io-app
# fix: [IOBP-126] Rename AUTHORIZED transaction status

## Short description
This PR renames the "AUTHORIZED" transaction status from "In corso" to "Autorizzata"

## List of changes proposed in this pull request
- Locale files for IT and EN languages;

## How to test
Into IDPay initiative transaction timeline, press on the first transaction with `io-dev-api-server` started and see the transaction status label to "Autorizzata"
